<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
  
  


  <head>
    <title>
      float128GccNvcc.patch on Ticket #11852 – Attachment
     – Boost C++ Libraries
    </title>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
        <link rel="search" href="/trac/boost/search" />
        <link rel="help" href="/trac/boost/wiki/TracGuide" />
        <link rel="alternate" href="/trac/boost/raw-attachment/ticket/11852/float128GccNvcc.patch" type="text/x-diff; charset=utf-8" title="Original Format" />
        <link rel="up" href="/trac/boost/ticket/11852" title="Ticket #11852" />
        <link rel="start" href="/trac/boost/wiki" />
        <link rel="stylesheet" href="/htdocs/common/css/trac.css" type="text/css" /><link rel="stylesheet" href="/htdocs/common/css/diff.css" type="text/css" /><link rel="stylesheet" href="/htdocs/common/css/code.css" type="text/css" />
        <link rel="shortcut icon" href="https://svn.boost.org/htdocs/site/favicon.ico" type="image/x-icon" />
        <link rel="icon" href="https://svn.boost.org/htdocs/site/favicon.ico" type="image/x-icon" />
      <link type="application/opensearchdescription+xml" rel="search" href="/trac/boost/search/opensearch" title="Search Boost C++ Libraries" />
    <script type="text/javascript" src="/htdocs/common/js/jquery.js"></script><script type="text/javascript" src="/htdocs/common/js/babel.js"></script><script type="text/javascript" src="/htdocs/common/js/trac.js"></script><script type="text/javascript" src="/htdocs/common/js/search.js"></script><script type="text/javascript" src="/htdocs/common/js/diff.js"></script>
    <!--[if lt IE 7]>
    <script type="text/javascript" src="/htdocs/common/js/ie_pre7_hacks.js"></script>
    <![endif]-->
      <script type="text/javascript" src="/htdocs/common/js/folding.js"></script>
      <script type="text/javascript">
        jQuery(document).ready(function($) {
          $('#preview table.code').enableCollapsibleColumns($('#preview table.code thead th.content'));
        });
      </script>
  </head>
  <body>
    <div id="banner">
      <div id="header">
        <a id="logo" href="https://svn.boost.org/trac/boost"><img src="/htdocs/site/boost.png" alt="C++ Boost" /></a>
      </div>
      <form id="search" action="/trac/boost/search" method="get">
        <div>
          <label for="proj-search">Search:</label>
          <input type="text" id="proj-search" name="q" size="18" value="" />
          <input type="submit" value="Search" />
        </div>
      </form>
      <div id="metanav" class="nav">
    <ul>
      <li class="first"><a href="/trac/boost/login">Login</a></li><li><a href="/trac/boost/wiki/TracGuide">Help/Guide</a></li><li><a href="/trac/boost/about">About Trac</a></li><li class="last"><a href="/trac/boost/prefs">Preferences</a></li>
    </ul>
  </div>
    </div>
    <div id="mainnav" class="nav">
    <ul>
      <li class="first"><a href="/trac/boost/wiki">Wiki</a></li><li><a href="/trac/boost/timeline">Timeline</a></li><li><a href="/trac/boost/roadmap">Roadmap</a></li><li><a href="/trac/boost/report">View Tickets</a></li><li><a href="/trac/boost/newticket">New Ticket</a></li><li class="last"><a href="/trac/boost/search">Search</a></li>
    </ul>
  </div>
    <div id="main">
      <div id="ctxtnav" class="nav">
        <h2>Context Navigation</h2>
          <ul>
              <li class="last first"><a href="/trac/boost/ticket/11852">Back to Ticket #11852</a></li>
              <li class="last">Server Load Average:
          <b>0.34
</b>
              </li>
          </ul>
        <hr />
      </div>
    <div id="content" class="attachment">
        <h1><a href="/trac/boost/ticket/11852">Ticket #11852</a>: float128GccNvcc.patch</h1>
        <table id="info" summary="Description">
          <tbody>
            <tr>
              <th scope="col">File float128GccNvcc.patch,
                <span title="552 bytes">552 bytes</span>
                (added by a.huebl@…, <a class="timeline" href="/trac/boost/timeline?from=2015-12-18T04%3A22%3A02-05%3A00&amp;precision=second" title="2015-12-18T04:22:02-05:00 in Timeline">4 months</a> ago)</th>
            </tr>
            <tr>
              <td class="message searchable">
                <p>
Patch to avoid float128 with nvcc
</p>

              </td>
            </tr>
          </tbody>
        </table>
        <div id="preview" class="searchable">
          
  <div class="diff">
  <ul class="entries">
      <li class="entry">
        <h2>
          <a>include/boost/config/compiler/gcc.hpp</a>
        </h2>
        <pre> include/boost/config/compiler/gcc.hpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/boost/config/compiler/gcc.hpp b/include/boost/config/compiler/gcc.hpp
index d9dd59d..48605eb 100644</pre>
        <table class="trac-diff inline" summary="Differences" cellspacing="0">
              <colgroup><col class="lineno" /><col class="lineno" /><col class="content" /></colgroup>
              <thead>
                <tr>
                  <th title="File a/include/boost/config/compiler/gcc.hpp">
                      a
                  </th>
                  <th title="File b/include/boost/config/compiler/gcc.hpp">
                      b
                  </th>
                  <td><em></em> </td>
                </tr>
              </thead>
            <tbody class="unmod">
                  <tr>
                          <th>153</th><th>153</th><td class="l"><span>#else</span> </td>
                  </tr><tr>
                          <th>154</th><th>154</th><td class="l"><span>#include &lt;stddef.h&gt;</span> </td>
                  </tr><tr>
                          <th>155</th><th>155</th><td class="l"><span>#endif</span> </td>
                  </tr>
            </tbody><tbody class="mod">
                      <tr class="first">
                        <th>156</th><th> </th><td class="l"><span>#if defined(_GLIBCXX_USE_FLOAT128) &amp;&amp; !defined(__STRICT_ANSI__)<del></del></span> </td>
                      </tr>
                      <tr class="last">
                        <th> </th><th>156</th><td class="r"><span>#if defined(_GLIBCXX_USE_FLOAT128) &amp;&amp; !defined(__STRICT_ANSI__)<ins>&nbsp;&amp;&amp; !defined(__CUDACC__)</ins></span> </td>
                      </tr>
            </tbody><tbody class="unmod">
                  <tr>
                          <th>157</th><th>157</th><td class="l"><span># define BOOST_HAS_FLOAT128</span> </td>
                  </tr><tr>
                          <th>158</th><th>158</th><td class="l"><span>#endif</span> </td>
                  </tr><tr>
                          <th>159</th><th>159</th><td class="l"><span></span> </td>
                  </tr>
            </tbody>
        </table>
      </li>
  </ul>
</div>

        </div>
    </div>
    <div id="altlinks">
      <h3>Download in other formats:</h3>
      <ul>
        <li class="last first">
          <a rel="nofollow" href="/trac/boost/raw-attachment/ticket/11852/float128GccNvcc.patch">Original Format</a>
        </li>
      </ul>
    </div>
    </div>
    <div id="footer" lang="en" xml:lang="en"><hr />
      <a id="tracpowered" href="http://trac.edgewall.org/"><img src="/htdocs/common/trac_logo_mini.png" height="30" width="107" alt="Trac Powered" /></a>
      <p class="left">
        Powered by <a href="/trac/boost/about"><strong>Trac 0.12.2</strong></a><br />
        By <a href="http://www.edgewall.org/">Edgewall Software</a>.
      </p>
      <p class="right">Visit the Trac open source project at<br /><a href="http://trac.edgewall.org/">http://trac.edgewall.org/</a></p>
    </div>
  
<!-- Google Analytics Widget Code -->
<script type="text/javascript">
  var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
  document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
  try {
    jQuery(document).ready( function() {
      var pageTracker = _gat._getTracker("UA-15160295-3");
      pageTracker._setDomainName(".boost.org");
      pageTracker._setAllowLinker()
      jQuery('a').each( function() {
        var url;
        if ( this.href.match("http:\/\/svn.boost.org") ) {
          if ( this.href.match(/\.(zip|tar|tar.gz|tar.bzip|egg)$/) ) {
            url = this.pathname + this.search;
          };
        } else {
          var port = '';
          if ( this.port != '') port = ':'+this.port;
          url = '/external/' + this.hostname + port + this.pathname + this.search;
        };
        if ( url ) {
          if ( ! url.match('^\/attachment\/') ) {
            jQuery(this).click( function() {
              pageTracker._trackPageview(url);
            });
          };
        };
      });
    pageTracker._trackPageview();
    });
  } catch(err) {}
</script>
<!-- Google Analytics Widget Code Ended -->
</body>
</html>